src/Testing/testProcessCommandLine.cpp

```diff
@@ -133,11 +133,21 @@ TEST_CASE("Cipher type declared with Caesar cipher")
 
 TEST_CASE("Cipher type declared with Playfair cipher")
 {
-    ProgramSettings settings{false, false, "", "", {}, {}, CipherMode::Encrypt};
+    ProgramSettings settings{false, false, "", "", {}, {}, CipherMode::Encrypt, {}};
     const std::vector<std::string> cmdLine{"mpags-cipher", "-c", "playfair"};
     const bool res{processCommandLine(cmdLine, settings)};
 
     REQUIRE(res);
     REQUIRE(settings.cipherType.size() == 1);
     REQUIRE(settings.cipherType[0] == CipherType::Playfair);
+}
+
+TEST_CASE("Multi-cipher option processed correctly", "[commandline]")
+{
+    ProgramSettings settings{false, false, "", "", {}, {}, CipherMode::Encrypt, {}};
+    const std::vector<std::string> cmdLine{"mpags-cipher", "--multi-cipher", "3"};
+    const bool res{processCommandLine(cmdLine, settings)};
+
+    REQUIRE(res);
+    REQUIRE(settings.nExpectedCiphers == 3);
```

Comment:
There are a few more test cases that could be added here.  Firstly, you need a test for the `-c vigenere` case.  Then for `--multi-cipher` you should check for correct handling of a missing subsequent argument, an invalid subsequent argument, and finally what happens if the number passed here is different from the number of ciphers that are actually specified.  In fact, that is the case here, so does this test pass?